website/docs/docs/build/metricflow-commands.md

```diff
@@ -580,6 +582,26 @@ dbt sl query --metrics revenue --group-by metric_time__month # In dbt Cloud
 mf query --metrics revenue --group-by metric_time__month # In dbt Core
 ```
 
+### Exports
+
+Run [exports for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use this command to test and generate exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.
+
+```bash
+dbt sl export # In dbt Cloud
+
+mf export # In dbt Core
```

Comment:
This isn't a thing in OSS, as exports are a cloud limited feature

Reply:
Oh yes, great shout @WilliamDee !